src/AuthenticatedAPIClient/AccessToken.js

```diff
@@ -0,0 +1,91 @@
+import Cookies from 'universal-cookie';
+import jwtDecode from 'jwt-decode';
+import axios from 'axios';
+import { logInfo, logError } from '@edx/frontend-logging';
+
+const httpClient = axios.create();
+const cookies = new Cookies();
+
+
+const getCookieValue = (name) => {
+  try {
+    return cookies.get(name);
+  } catch (error) {
+    logInfo(`Error reading the access token cookie: ${name}.`);
+    return null;
+  }
+};
+
+const decodeJwtCookie = (cookieValue) => {
+  try {
```

Comment:
Do we want to assert that cookieValue is not empty here, since that is the assumption.  Otherwise, someone could start writing lots of `'Error decoding JWT token.'` logs (like we used to), when we never should have tried to decode.